internal/operators/ocs/validations.go

```diff
@@ -69,24 +71,25 @@ func setOperatorStatus(cluster *models.Cluster, status string) error {
 func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
 	var status string
 	hosts := cluster.Hosts
-
+	var err error
 	if int64(len(hosts)) < o.OCSRequiredHosts {
 		status = "Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. "
 		err := setOperatorStatus(cluster, status)
```

Comment:
this is a different err (not the one declared on line 74. Is that what was meant?

Reply:
fixed